Consolidate/update QCA pages (#5072)
Qiskit/documentation: docs/guides/qiskit-code-assistant.mdx

Reviewer: this section (`Quickstart (recommended)`) is no longer relevant and should be removed. It is just instructions for the older extensions. Maybe replace it with a link to below section which has the local install instructions?

Reviewer: ```suggestion
### Local setup

To configure your local setup , expand the sections below.
```

Reviewer: update model's name
```suggestion
2. [Qiskit/Qwen2.5-Coder-14B-Qiskit](https://huggingface.co/Qiskit/Qwen2.5-Coder-14B-Qiskit) - Released June 2025
```

Reviewer: update model's name
```suggestion
2. [Qiskit/Qwen2.5-Coder-14B-Qiskit-GGUF](https://huggingface.co/Qiskit/Qwen2.5-Coder-14B-Qiskit-GGUF) – Released **June 2025**  
```

Reviewer: update model's name
```suggestion
| Qwen2.5-Coder-14B-Qiskit | 25.17 | 49.01 | 91.46 | 4.21 | 53.90 | 97.00 | 77.60 | 49.64 | 65.18 | 37.82 | June 2025 | 2.0 |
```

description: Learn how to use Qiskit Code Assistant, a generative AI code assistant.
---

{/* cspell:ignore ollama, gguf, modelfile, safetensors, winget, MBPP, opencode */}

# Qiskit Code Assistant

> Qiskit Code Assistant aims to make quantum computing more accessible to new Qiskit adopters and to improve the coding experience for current users. It is trained using millions of text tokens from Qiskit SDK, years of Qiskit code examples, and IBM Quantum&reg; features. Qiskit Code Assistant can help your quantum development workflow by offering LLM-generated suggestions based on [IBM Granite models](https://www.ibm.com/products/watsonx-ai/foundation-models), which incorporate the latest features and functionalities from IBM&reg;.

Reviewer 2: Since there will be provided only Qiskit Code Assistant-LLMs (no dedicated extensions or service), should re rephrase the beginning of the paragraph?

> Qiskit Code Assistant aims to make quantum computing more accessible to new Qiskit adopters and to improve the coding experience for current users. It is trained using millions of text tokens from Qiskit SDK, years of Qiskit code examples, and IBM Quantum&reg; features. Qiskit Code Assistant can help your quantum development workflow by offering LLM-generated suggestions based on [IBM Granite models](https://www.ibm.com/products/watsonx-ai/foundation-models), which incorporate the latest features and functionalities from IBM&reg;.

Reviewer 2: ```suggestion
Qiskit Code Assistant LLMs aim to make quantum computing more accessible to new Qiskit adopters and to improve the coding experience for current users. It is trained using millions of text tokens from Qiskit SDK, years of Qiskit code examples, and IBM Quantum&reg; features. Qiskit Code Assistant can help your quantum development workflow by offering LLM-generated suggestions based on [IBM Granite and other open-source models](https://huggingface.co/Qiskit), which incorporate the latest features and functionalities from IBM&reg;.
```

> To skip to the installation instructions, go to the [Install Qiskit Code Assistant](#install) section.

Reviewer: should this link to the Local Setup section which actually has the instructions? also could this be more prominent as to not be easily/missed glanced over?

The datasets for Qiskit HumanEval and Qiskit HumanEval Hard are available at these websites: [Qiskit HumanEval](https://huggingface.co/datasets/Qiskit/qiskit_humaneval) and [Qiskit HumanEval](https://huggingface.co/datasets/Qiskit/qiskit_humaneval_hard). You can contribute to the development of these benchmarks at the [GitHub repository](https://github.com/qiskit-community/qiskit-human-eval).

<span id="install"></span>
## Install Qiskit Code Assistant

> Learn how to install, configure, and use any of Qiskit Code Assistant models on your local machine.

Reviewer: related to the [above comment](https://github.com/Qiskit/documentation/pull/5072/changes#r3243006850), there is no instructions here. should it link to the Local Setup section?

> ### Available models

Reviewer: maybe make this a top level section (instead of subsection of install)

```suggestion
## Available models
```
